config/routes.rb

```diff
@@ -54,6 +54,11 @@
   get "/terms_and_conditions", to: "static_pages#terms_and_conditions"
   get "/cookie_policy", to: "static_pages#cookie_policy"
 
+  resources :adopter_applications, path: "applications",
```

Comment:
These two resource routes should not be added here - they have been moved under organization scope. Looks like these came from an old branch?